internal/util/reports.go

```diff
@@ -10,8 +11,10 @@ import (
 
 var (
 	reportsHost  = "kong-hf.konghq.com"
-	reportsPort  = 61829
+	reportsPort  = 61833
 	pingInterval = 3600
+	tlsConf      = tls.Config{MinVersion: tls.VersionTLS12}
```

Comment:
While we probably presume the endpoint will be negotiating `TLS1.3` by default and we may also assume that would never change in a negative direction, it seems we should simply require `tls.VersionTLS13` here anyway, no?
```suggestion
	tlsConf      = tls.Config{MinVersion: tls.VersionTLS13}
```

Reply:
Dunno. I defaulted to the oldest version without significant currently known flaws because it is a minimum and the client will still use 1.3 opportunistically if available.

The upstream does support it, so the only issue I can think of are if there are instances behind intercept proxies that haven't implemented 1.3 yet. Asked to see if we have version data from the Kong reporters anywhere, but I doubt it. Absent that we can probably use 1.3 and revert if we see an unusual drop in reporting instances.